src/middleware/errorhandler.middleware.js

```diff
@@ -0,0 +1,12 @@
+// eslint-disable-next-line no-unused-vars
+const ErrorHandler = (err, req, res, next) => {
+  const errStatus = err.statusCode || 500;
+  const errMsg = err.message || 'Internal Server Error';
+  res.status(errStatus).json({
+    code: errStatus,
+    message: errMsg,
+    error: err,
+  });
+};
+
+export default ErrorHandler;
```

Comment:
why are you disabling the eslint

Reply:
Yes, express error handler need to have 4 parameter, or else its just another middleware